CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# v1.8.0:
+
+### Features:
+
+- [Feature #35](https://github.com/google/osv-scanner/issues/35)
+  OSV-Scanner now scans transitive dependencies in Maven `pom.xml` files!
+  See [our documentation](https://google.github.io/osv-scanner/supported-languages-and-lockfiles/#transitive-dependency-scanning) for more information.
+- [Feature #944](https://github.com/google/osv-scanner/pull/944)
+  The `osv-scanner.toml` configuration file can now filter specific packages with new `[[PackageOverrides]]` sections:
+  ```toml
+  [[PackageOverrides]]
+  name = "lib"
+  version = "1.0.0"
+  ecosystem = "Go"
+  ignore = true
+  # effectiveUntil = 2022-11-09 # Optional exception expiry date
+  reason = "abc"
+  license.override = ["MIT"]
+  ```
+
+### Minor Updates
+
+- [Feature #1039](https://github.com/google/osv-scanner/pull/1039) Renamed the `--experimental-local-db` flag to `--experimental-download-offline-databases`.
```

Comment:
```suggestion
- [Feature #1039](https://github.com/google/osv-scanner/pull/1039) The`--experimental-local-db` flag has been removed, and a new flag `--experimental-download-offline-databases` is added to better reflect what `--experimental-local-db` is doing. 

To replicate the `--experimental-local-db` flag, use: `--experimental-offline --experimental-download-offline-databases`. This will run osv-scanner in offline mode, but download the latest version of the vulnerability databases before scanning.
```

Reply:
I've reworded this suggestion slightly